server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+
+const app = express()
+
+app.get("/ping",(req , res)=>{
```

Comment:
Consider adding a catch-all route to handle undefined routes and return a 404 error. [medium]